buildkite/scripts/test.sh

```
@@ -12,6 +12,8 @@ set_environment_variables() {
     export AWS_SECRET_ACCESS_KEY=$(buildkite-agent secret get AWS_SECRET_ACCESS_KEY)
     export BUILDKITE_ANALYTICS_TOKEN=$(buildkite-agent secret get TESTSUITE_TOKEN)
     export AWS_REGION=eu-west-1
+    export DOCKER_USERNAME=$(buildkite-agent secret get DOCKER_USERNAME)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)